tex/pgfcircbipoles.tex

```diff
@@ -255,7 +255,7 @@
 {bmultiwire}
 {\ctikzvalof{bipoles/multiwire/height}}
 {\ctikzvalof{bipoles/multiwire/width}}
-{
+{blocks
```

Comment:
This is strange --- I think it's a typo? (BTW, thanks for the PR; will review as soon as I can).

Reply:
I cannot explain this one as did mostly copy paste. 
deleted it anyways, thx!

Reply:
Ok!